src/user/ImageMaker.ts

```diff
@@ -74,7 +75,7 @@ export default class ImageMaker {
     constructor(
         private dockerRegistryHelper: DockerRegistryHelper,
         private dockerApi: DockerApi,
-        private namespace: string,
+        private dataStore: DataStore,
```

Comment:
Only pass the env vars as needed - avoid adding  an unnecessary extra dependency on DataStore.